ozhera-prometheus-agent/ozhera-prometheus-agent-common/src/main/java/com/xiaomi/youpin/prometheus/agent/util/DateUtil.java

```diff
@@ -27,4 +27,25 @@ public static String TimeStampToISO8601UTC(long timeStamp) {
         return utcTime;
     }
 
+    @SneakyThrows
+    public static long ISO8601UTCTOTimeStamp(String utcTime) {
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
+        Date date = sdf.parse(utcTime);
+        return date.getTime();
+    }
+
+    @SneakyThrows
+    public static String ISO8601UTCTOCST(String utcTime) {
+        SimpleDateFormat sdfUTC = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```

Comment:
SimpleDateFormat may have concurrency issues in concurrent situations. Do you need to pay attention to this

Reply:
There should be no concurrency issues involved here